cpp/src/replace/replace.cu

```diff
@@ -142,7 +149,11 @@ namespace{ //anonymous
 
 template <typename Type>
 __global__
-void replace_nulls_with_scalar(gdf_size_type size, Type* out_data, gdf_valid_type * out_valid, const Type *in_data_scalar) 
+  void replace_nulls_with_scalar(gdf_size_type size,
+                                 const Type* in_data,
+                                 const bit_mask_t* in_valid,
```

Comment:
```suggestion
                                 const bit_mask_t* const __restrict__ in_valid,
```

Reply:
Shouldn't `in_data` be `__restrict__` too?